app/views/devise/sessions/new.html.slim

```diff
@@ -17,21 +17,21 @@
           .form-group
             label.col-md-3.control-label for='password'
             .col-md-9
-              .input-group style=('margin-bottom: 25px') 
+              .input-group style=('margin-bottom: 25px')
                 span.input-group-addon
-                  i.fa.fa-briefcase aria-hidden='true' 
+                  i.fa.fa-briefcase aria-hidden='true'
                 = f.password_field :password, autofocus: true, autocomplete: 'off', class: 'form-control', placeholder: 'Password'
           .input-group
             .checkbox
               label
                 input#login-remember name='remember' type='checkbox' value='1' /
                 | Remember me
-          .form-group style='margin-top:10px' 
+          .form-group style='margin-top:10px'
             .col-sm-12.controls
               = f.button 'Sign in', class: 'btn btn-success', data: { disable_with: "<i class='fa fa-circle-o-notch fa-spin' style='font-size:24px'></i> Signin in..." }
-              a#btn-fblogin.btn.btn-primary href='#'  Login with Facebook
+              = link_to "Sign in with Facebook", user_facebook_omniauth_authorize_path, class: 'btn btn-primary ml-2'
           .form-group
             .col-md-12.control
-              div style=('border-top: 1px solid#888; padding-top:15px; font-size:85%') 
+              div style=('border-top: 1px solid#888; padding-top:15px; font-size:85%')
                 | Don't have an account!
```

Comment:
Create a new PR to move all static text to locale file.
http://rusrails.ru/rails-internationalization-i18n-api#lenivyy-poisk